plugins/inputs/snmp/snmp.go

```diff
@@ -535,8 +538,8 @@ func (t Table) Build(gs snmpConnection, walk bool) (*RTable, error) {
 		Time: time.Now(), //TODO record time at start
 		Rows: make([]RTableRow, 0, len(rows)),
 	}
-	for _, r := range rows {
-		if len(r.Tags) < tagCount {
+	for i, r := range rows {
+		if len(r.Tags)+emptyTags[i] < tagCount {
 			// don't add rows which are missing tags, as without tags you can't filter
 			continue
 		}
```

Comment:
Tagless measurements are already allowed into InfluxDB. I'm not sure I understand why the SNMP plugin would be asserting otherwise.

But if I understand correctly, @phemmer you're just saying that in that case this change is overly complicated? we can just remove any tag-checking at all rather than counting and incrementing this "emptyTags" map?